mayastor/src/bdev/nexus/nexus_label.rs

```diff
@@ -462,7 +496,9 @@ impl GPTHeader {
             deserialize_from(&mut reader).context(DeserializeError {})?;
 
         if gpt.header_size != 92 {
-            return Err(LabelError::GptHeaderSize {});
+            return Err(LabelError::GptHeaderSize {
+                size: 92,
```

Comment:
would we want to say here that the expected size (92) is not the gpt.header_size?